apis/chatbotapi/index.html

```diff
@@ -0,0 +1,25 @@
+---
+---
+<!DOCTYPE html>
+<html>
+	<head>
+		{% include head.html %}
+		<meta name="robots" content="index, follow">
+		<script src="https://rebilly.github.io/ReDoc/releases/v1.19.1/redoc.min.js"> </script>
```

Comment:
this is a wrong script URL! check the api-documentation branch contentapis index.html and update accordingly.